index.md

```diff
@@ -128,221 +128,172 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
+|   | reference/amc-command-reference/info | [AMC command reference - info](https://discourse.ubuntu.com/t/40784) |
+|   | reference/amc-command-reference/init | [AMC command reference - init](https://discourse.ubuntu.com/t/40785) |
+|   | reference/amc-command-reference/launch | [AMC command reference - launch](https://discourse.ubuntu.com/t/40786) |
+|   | reference/amc-command-reference/list | [AMC command reference - list](https://discourse.ubuntu.com/t/40787) |
+|   | reference/amc-command-reference/logs | [AMC command reference - logs](https://discourse.ubuntu.com/t/40788) |
+|   | reference/amc-command-reference/node | [AMC command reference - node](https://discourse.ubuntu.com/t/40789) |
+|   | reference/amc-command-reference/remote | [AMC command reference - remote](https://discourse.ubuntu.com/t/40790) |
+|   | reference/amc-command-reference/shell | [AMC command reference - shell](https://discourse.ubuntu.com/t/40791) |
+|   | reference/amc-command-reference/show-log | [AMC command reference - show-log](https://discourse.ubuntu.com/t/40792) |
+|   | reference/amc-command-reference/show | [AMC command reference - show](https://discourse.ubuntu.com/t/40793) |
+|   | reference/amc-command-reference/start | [AMC command reference - start](https://discourse.ubuntu.com/t/40794) |
+|   | reference/amc-command-reference/stop | [AMC command reference - stop](https://discourse.ubuntu.com/t/40795) |
+|   | reference/amc-command-reference/wait | [AMC command reference - wait](https://discourse.ubuntu.com/t/40796) |
 [/details]
 
 ## Redirects
 
 [details=Mapping table]
 | Path | Location |
 | ---- | -------- |
-| /docs/install-appliance | /docs/tut/installing-appliance |
-| /docs/lxd-auto-scaling | /docs/exp/clustering |
-| /docs/usage/usecase-streaming-sdk | /docs/ref/sdks |
-| /docs/reference/ams-sdk-api-reference | /docs/ref/sdks |
-| /docs/installation/installation-appliance | /docs/tut/installing-appliance |
-| /docs/manage/getting-started | /docs/tut/getting-started |
-| /docs/installation/installation-quickstart | /docs/howto/install/deploy-juju |
-| /docs/installation/installation-customizing | /docs/howto/install/customise |
-| /docs/installation/high-availability | /docs/howto/install/high-availability |
-| /docs/installation/validation | /docs/howto/install/validate |
-| /docs/installation/upgrading-from-previous-versions | /docs/howto/update/upgrade-anbox |
-| /docs/howto/manage/managing-images | /docs/howto/manage/images |
-| /docs/howto/manage/managing-addons | /docs/howto/addons/landing |
-| /docs/howto/manage/managing-logs | /docs/howto/manage/logs |
-| /docs/howto/manage/managing-ams-access | /docs/howto/manage/ams-access |
-| /docs/manage/managing-ams-access | /docs/howto/manage/ams-access |
-| /docs/installation/installation-application-registry | /docs/howto/aar/deploy |
-| /docs/usage/usecase-application-testing | /docs/howto/application/test |
-| /docs/usage/usecase-virtual-devices | /docs/howto/application/virtual-devices |
```

Comment:
Shouldn't we still keep these redirects, just in case somebody still has old bookmarks or references?

Reply:
Yes, I was initially thinking of keeping them but the redirects table keeps growing too long and these redirects are at least a year before (or closer to that range). So I thought it makes much more sense to keep the recent redirects. Maybe we can discuss this in the SU and see what others think as well?

Reply:
Yes, I was initially thinking of keeping them but the redirects table keeps growing too long and these redirects are at least a year before (or closer to that range). So I thought it makes much more sense to keep the recent redirects. Maybe we can discuss this in the SU and see what others think as well?